TrackletAlgorithm/VMStubMEMemoryCM.h

```diff
@@ -110,6 +111,17 @@ class VMStubMECM : public VMStubMECMBase<VMSMEType>
     return data_.range(kVMSMEBendMSB,kVMSMEBendLSB);
   }
 
+  bool isPSStub() const {
+    if(VMSMEType == BARRELPS) return true;
+    else if(VMSMEType == BARREL2S) return false;
+    else if(VMSMEType == DISK) return (getBend() & (1 << (VMStubMECMBase<VMSMEType>::kVMSMEBendSize-1))) == 0 && getBend() != 0; // Check highest 1 bits regardless of template type
```

Comment:
Can you explain how this works?

Reply:
I don't see how this can work. If you find a 1 in the highest bit you know that it has to be a PS module, but if you find a zero you don't  know if it is PS or 2S it can just be a 2S module with a bend less than 8. To determine if it is a PS or 2S stub you can use the rzbin and the finerz position.

Reply:
Turns out this is just stray code. The only `isPSStub` method I use is the one in the `AllStub` memory. I've removed this one.